### ISSUE TITLE
[BUG] Browser selection colours are low contrast for some colour palettes

### ISSUE THREAD

**Issue by Jermolene** [Jermolene/TiddlyWiki5#4846]
**Describe the bug**
In #4590 we added CSS to change the browser selection colours to use the palette:

```
::selection {
	background-color: <<colour primary>>;
	color: <<colour page-background>>;
}

::-moz-selection {
	background-color: <<colour primary>>;
	color: <<colour page-background>>;
}
```

The problem is that some of the palettes that ship with the core do not have good contrast between the primary colour and the page background colour. For example, this is how selected text looks with the "Dark Photos" palette:

![image](https://user-images.githubusercontent.com/174761/92725005-60d36500-f363-11ea-9eac-02bb68dc747b.png)

**To Reproduce**
1. Go to https://tiddlywiki.com/prerelease
2. Use the control panel to switch to the "Dark Photos" palette
3. Select some text on the page and observe the colour scheme

**Expected behavior**
Selected text should always be legible with all palettes.

**Comment by twMat** [comment]
Since you post this, I'm probably about to say something either trivial or irrelevant here but... what about `mix-blend-mode`? ([examples](https://css-tricks.com/methods-contrasting-text-backgrounds/))

**Comment by Jermolene** [comment]
Hi @twMat part of the problem is that modern browsers use the operating system settings for text selection, and use heuristics to ensure the selection is always visible. So, for most users, the browser defaults are probably better.

But the main problem is that #4590 hardcoded the selection to be coloured with the primary colour and the page background colour, which are not guaranteed to have good contrast because they are not used together in the core UI.

**Comment by Jermolene** [comment]
Fixed in 543f9107b6a9de5fe13542fa30d9a4483d8b3eb8
